src/js/track.js

```diff
@@ -0,0 +1,83 @@
+var _ = require('underscore');
+var d3 = require('d3');
+var $ = require('jquery');
+var D3SVGCellRenderer = require('./d3_svg_cell_renderer');
+var utils = require('./utils');
+
+module.exports = {};
+
+var defaultTrackConfig = {
+	label: 'Gene',
+	id_member: 'sample',
+	cell_height: 20,
+	track_height: 20,
+	track_padding: 2.5,
+}; 
+
+function Track(name, oncoprint, data, config) {
+	var self = this;
+	self.name = name;
+	self.config = $.extend({}, defaultTrackConfig, config || {}); // inherit from default
```

Comment:
I prefer code over comments and I'm not the only one: http://www.pauloortins.com/2013/07/11/5-reasons-to-avoid-code-comments/. That article also has references to some pretty well-known code style books, which I have to admit I haven't read more than bits and pieces of.